content/en/docs/concepts/services-networking/connect-applications-service.md

```diff
@@ -394,8 +394,8 @@ kubectl edit svc my-nginx
 kubectl get svc my-nginx
 ```
 ```
-NAME       TYPE        CLUSTER-IP     EXTERNAL-IP        PORT(S)               AGE
-my-nginx   ClusterIP   10.0.162.149   162.222.184.144    80/TCP,81/TCP,82/TCP  21s
+NAME       TYPE           CLUSTER-IP     EXTERNAL-IP        PORT(S)               AGE
+my-nginx   LoadBalancer   10.0.0.216     xx.xxx.xxx.xxx     8080:30163/TCP        21s
```

Comment:
I think the ClusterIP should remain the same as in earlier examples.
```suggestion
my-nginx   LoadBalancer   10.0.162.149     xx.xxx.xxx.xxx     8080:30163/TCP        21s
```

Reply:
@Vageesha17 would you be willing to revise in line with https://github.com/kubernetes/website/pull/20219/files#r411816994 ?